### DISCUSSION TITLE
Handling hundreds of visitors at once

### DISCUSSION THREAD

**Discussion by github-user-77** [mozilla/hubs#3263] [Hubs Cloud Questions] [answered]
Hi There,
On our Hubs Cloud, we're expecting hundreds of people will turn up for the launch. There is a "foyer" room, which links off to about 20 different rooms.  There will first be speeches in the foyer for the launch, after which visitors will explore the 20 rooms.

What is the best way to handle the speeches etc? 

If we allow, say, 48 people in the foyer room(possible?), with 75 in the lobby of the foyer room, can we stream a feed into other overflow rooms, each with the same setup (48 in-room, 75 in lobby)?

If so, can we automate this – does Hubs know when limits have been reached? – or will we need to monitor and direct people to different rooms?

Alternatively (this is probably out of the question) is it possible to have the same users (i.e., the people giving the speeches) actively logged in to multiple rooms at once?

We're using T3.2xLarge.

**Comment by pattersonbl2** [comment, marked as answer]
This request would exceed the limit.  it would be 25 people to a room 75 in lobby area.  I am not sure if this question still need but feel free to reach out if you still need help with anything.
